gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -266,9 +273,27 @@ def _get_vendor_data(self, device_data, image_name):
             return None
         try:
             metadata = self.compute_handler_nova.get_image_metadata(
-                    token,
-                    self._get_admin_tenant_id(token=token),
-                    image_name)
+                token,
+                self._get_admin_tenant_id(token=token),
+                image_name)
+        except Exception as e:
+            self._increment_stats_counter('image_details_get_failures')
+            LOG.error(_LE('Failed to get image metadata for image '
+                          'name: %(image_name)s. Error: %(error)s'),
+                      {'image_name': image_name, 'error': e})
+            return None
+        vendor_data = self._verify_vendor_data(image_name, metadata)
+        if not vendor_data:
+            return None
+        return vendor_data
+
+    def _get_vendor_data_v1(self, token,
```

Comment:
same comment w.r.t duplicate functions
_get_vendor_data_v1() function is almost similar with _get_vendor_data(),